src/Sulu/Bundle/AdminBundle/Navigation/ContentNavigationItem.php

```diff
@@ -241,7 +241,7 @@ public function toArray()
             'action' => $this->getAction(),
             'display' => $this->getDisplay(),
             'component' => $this->getComponent(),
-            'componentOptions' => $this->getComponentOptions(),
+            'componentOptions' => array_merge($this->getComponentOptions(), $options),
```

Comment:
Not without some thought at least -- there is no obvious way that I am aware of.